README.md

```diff
@@ -4,110 +4,282 @@
 
 This Javascript-based Library allows an education provider to integrate web-based external Tools into their chosen learning Platform, such as Moodle, Blackboard, Edgenuity, etc, without needing to understand the underlying Learning Tools Interoperability (LTI) standards (https://www.imsglobal.org/activity/learning-tools-interoperability). 
 
-This Library supports the LTI 1.3 Core standards (https://www.imsglobal.org/spec/lti/v1p3), which implements up-to-date privacy and security standards.  It also provides limited support of the LTI Advantage Names and Roles Provisioning Services (https://www.imsglobal.org/spec/lti-nrps/v2p0/) and Assignment and Grade Services (https://www.imsglobal.org/spec/lti-ags/v2p0).
+This Library supports the LTI 1.3 Core standards (https://www.imsglobal.org/spec/lti/v1p3), which implements up-to-date privacy and security standards.  It also provides limited support the Assignment and Grade Services (https://www.imsglobal.org/spec/lti-ags/v2p0).
 
-In the future, this Library will be updated to fully support the LTI Advantage extensions (https://www.imsglobal.org/spec/lti/v1p3/impl) for Names/Roles, and Assignments/Grades.  At this time, LTI Advantage Deep Linking (https://www.imsglobal.org/spec/lti-dl/v2p0/) is not supported.
+In the future, this Library will be updated to fully support the LTI Advantage extensions (https://www.imsglobal.org/spec/lti/v1p3/impl) for Assignments/Grades.  At this time, LTI Advantage Names and Roles Provisioning Services (https://www.imsglobal.org/spec/lti-nrps/v2p0/) and Deep Linking (https://www.imsglobal.org/spec/lti-dl/v2p0/) is not supported.
 
 ## Overview
 
-Follow these steps to use this Library:
+Follow these steps to implement this Library:
 
+0. Develop a Tool
 1. Install Library
-2. Setup Routes
-3. Register Platform with Tool
+2. Setup MongoDB
+3. Setup Server and Routes
 4. Add Tool to Platform
+5. Register Platform with Tool
+6. Run your Server
 
 Optionally, you can:
 
-- Experiment with Example Tool
+- View the OIDC Tool Launch Flow
 - Use the Test Suite
 - View the Glossary
 
+### 0. Develop a Tool
+
+It is assumed if you are interested in integrating a Tool into a Platform that you have already developed a working Tool.  If not, Step 0 is to develop your Tool that will be dropped into a learning Platform.  If you are not at that point yet, you can use the Example Tool related to this Library.
```

Comment:
I think changing, the first sentence would provide a little more clarity:

"If you are interested in integrating a tool into an existing platform, you must have one developed and working.

i also suggest providing a link to the example tool repo as a reference for understanding how to build a custom tool.